client/extensions/zoninator/components/settings/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import Navigation from '../navigation';
+
+const Settings = ( {
+	children,
+	tab,
+	translate
+} ) => {
+	return (
+		<Main className={ 'zoninator__main' }>
```

Comment:
No need for the curlies; this is just a plain old string. But this is likely the cause of the linting issue and why CCI is failing. You can store the class name in a variable and that should silence the linter. It's a bit of a hack, but better than having CircleCI fail. See https://github.com/Automattic/wp-calypso/blob/master/client/extensions/wp-job-manager/components/settings/index.jsx#L43.